.github/scripts/generate-crd-docs/templates/gv_list.tpl

```diff
@@ -0,0 +1,23 @@
+{{- define "gvList" -}}
+{{- $groupVersions := . -}}
+{{- $groupVersion := index $groupVersions 0 -}}
+
+---
+title: {{ $groupVersion.Version }}
+description: Reference information for {{ $groupVersion.GroupVersionString }}
+weight: 100
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+<!-- markdownlint-disable -->
```

Comment:
I wanted to specifically only disable it for the auto generated files so that we still check other files in the future in those folders if we add some.